app/src/main/res/layout/view_now_playing.xml

```diff
@@ -0,0 +1,43 @@
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/src/main/res/layout/view_now_playing.xml#L1"></span>The resource <code>R.layout.view_now_playing</code> appears to be unused</td>
    </tr>
  </tbody>
</table>